@xen-orchestra/web-core/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@xen-orchestra/web-core",
+  "type": "module",
+  "version": "0.0.1",
+  "private": true,
+  "exports": {
+    "./*": {
+      "types": "./lib/*",
+      "import": "./lib/*"
+    },
+    "./eslint": {
+      "import": "./eslint.config.js"
+    }
+  },
+  "files": [
+    "lib"
+  ],
+  "scripts": {
+    "lint": "eslint",
+    "lint:fix": "eslint --fix"
+  },
+  "devDependencies": {
+    "@antfu/eslint-config": "^2.4.6",
+    "@fontsource/poppins": "^5.0.8",
+    "@fortawesome/fontawesome-common-types": "^6.5.1",
+    "@fortawesome/fontawesome-svg-core": "^6.5.1",
+    "@fortawesome/free-solid-svg-icons": "^6.5.1",
+    "@fortawesome/vue-fontawesome": "^3.0.5",
+    "@tsconfig/node18": "^18.2.2",
+    "@types/lodash-es": "^4.17.12",
+    "@types/node": "^18.19.3",
+    "@vitejs/plugin-vue": "^5.0.0-beta.1",
+    "@vue/tsconfig": "^0.5.1",
+    "@vueuse/core": "^10.7.0",
+    "eslint": "^8.56.0",
+    "glob": "^10.3.10",
+    "lodash-es": "^4.17.21",
+    "npm-run-all2": "^6.1.1",
+    "pinia": "^2.1.7",
+    "typescript": "~5.3.3",
+    "vite": "^5.0.10",
+    "vite-plugin-dts": "^3.6.4",
+    "vite-plugin-lib-inject-css": "^1.3.0",
+    "vue": "^3.4.0-beta.4",
+    "vue-router": "^4.2.5",
+    "vue-tsc": "^1.8.25"
+  },
+  "lint-staged": {
+    "*": "eslint --fix"
```

Comment:
Does lint-staged also run the rules defined in the root `package.json`?

https://github.com/lint-staged/lint-staged